src/app/update-toast/update-toast.component.ts

```diff
@@ -38,21 +38,28 @@ export class UpdateToastComponent implements OnInit {
   downloadingUpdate: boolean = false;
   updateDownloaded: boolean = false;
   version: string;
+  updateErrorSub: any;
+  updateError: boolean;
+  updateDownloadedSub: any;
```

Comment:
A reason these subs are "any" and no Subscription? also probs need to unsubscribe too